### PR TITLE
[BI-686] Remove method name and trait description

### PR DESCRIPTION
**Pull request by ctucker3**
Removes the method name and trait description from our endpoints and brapi communication.

### CODE REVIEW

**Review by timparsons — APPROVED** [review]

File: src/main/java/org/breedinginsight/daos/TraitDAO.java [hunk 1]
```diff
@@ -259,7 +258,6 @@ public TraitDAO(Configuration config, DSLContext dsl, BrAPIProvider brAPIProvide
                     .build();
             BrApiTrait brApiTrait = BrApiTrait.builder()
                     .traitName(trait.getTraitName())
-                    .traitDescription(trait.getDescription())
```
Comment (nickpalladino) [inline src/main/java/org/breedinginsight/daos/TraitDAO.java:262]: Similar comment as for the v1 observationvariables endpoint regarding autopopulation. Although I'm wondering what would happen in the case of an update. When we update in our other brapi endpoints we retrieve what's on in the brapi service then update fields and put that. Would we just overwrite those fields with autopopulated data?

**Review by nickpalladino — CHANGES_REQUESTED** [review]

File: src/main/java/org/breedinginsight/brapi/v1/services/BrapiObservationVariableService.java [hunk 2]
```diff
@@ -144,7 +143,6 @@ private ObservationVariable mapBiTraitToBrapiV1ObservationVariable(org.breedingi
                 //.alternativeAbbreviations(Arrays.asList(trait.getAbbreviations().clone()))
                 .attribute(trait.getAttribute())
                 .propertyClass(trait.getTraitClass())
-                .description(trait.getDescription())
```
[the remark this answers is not in the document]
Comment (ctucker3) [reply src/main/java/org/breedinginsight/brapi/v1/services/BrapiObservationVariableService.java:147]: Do you remember what they are being autopopulated from?

File: src/main/java/org/breedinginsight/daos/TraitDAO.java (hunk 1 above)
Comment (ctucker3) [reply src/main/java/org/breedinginsight/daos/TraitDAO.java:262]: I would think that we would adjust the value of the autopopulated fields as the fields that autopopulate them change.

File: src/main/java/org/breedinginsight/brapi/v1/services/BrapiObservationVariableService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickpalladino) [reply src/main/java/org/breedinginsight/brapi/v1/services/BrapiObservationVariableService.java:147]: From: https://breedinginsight.atlassian.net/wiki/spaces/BI/pages/629112866/Trait+Ontology+Template+-+Decisions+and+Feedback

- Trait Description field will be removed from template and UI - if needed for Field Book or BrAPI, will be auto-populated from Method Description.
- Method Name will be removed from template and UI - if needed for Field Book or BrAPI, will be auto-populated from Trait Name and Method Class.

File: src/main/java/org/breedinginsight/daos/TraitDAO.java (hunk 1 above)
Comment (nickpalladino) [reply src/main/java/org/breedinginsight/daos/TraitDAO.java:262]: Ya, makes sense.

File: src/main/java/org/breedinginsight/services/parsers/trait/TraitFileColumns.java [hunk 3]
```diff
@@ -25,11 +25,9 @@
     TRAIT_NAME("Trait name"),
     TRAIT_ABBREVIATIONS("Trait abbreviations"),
     TRAIT_SYNONYMS("Trait synonyms"),
-    TRAIT_DESCRIPTION("Trait description"),
```
[the remark this answers is not in the document]
Comment (ctucker3) [reply src/main/java/org/breedinginsight/services/parsers/trait/TraitFileColumns.java:28]: Update the download link. See the PR links below. I'll need @timparsons and @nickpalladino to approve these. 

bi-web: 
https://github.com/Breeding-Insight/bi-web/pull/59

requirements: 
https://github.com/Breeding-Insight/requirements/pull/13

File: src/main/java/org/breedinginsight/brapi/v1/services/BrapiObservationVariableService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ctucker3) [reply src/main/java/org/breedinginsight/brapi/v1/services/BrapiObservationVariableService.java:147]: updated and pushed

**Review by nickpalladino — APPROVED** [review]

**Pull request merged**